src/components/_inputs/js/InputNumber.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  name: 'InputNumber',
+  props: {
+    name: {
+      type: String,
+      required: true
+    },
+    minimum: {
+      type: Number,
+      default: -99999999
+    },
+    maximum: {
+      type: Number,
+      default: 99999999
+    },
+    default: {
+      type: Number
+    }
```

Comment:
Acho que você precisa de mais uma prop pra definir se é inteiro...

Reply:
@leomp12 Concordo, devo checar o tipo do campo certo? Se é "number" ou "inteiro", a minha dúvida é quanto casas decimais, sempre será 2? (Pergunto isso por conta de supermercados por exemplo, onde pode haver quantidade pesável que possui 3 casas decimais, mas acredito que isso não se enquadre em e-commerce, você poderá me dizer melhor.)

Reply:
@thiagoabreudev a princípio pode deixar sempre duas casas decimais quando for dinheiro, mas nem sempre quando for number será dinheiro, em outros casos pode deixar livre, só um input number sem definir o número de casas...

Reply:
Neste caso já existe o InputMoney  que possui 2 casas decimais. Vou deixar dessa forma por enquanto. Observei que no schema vem a definição de type como de "integer" e também "number" **(ex: min_amout com o type: Integer no schema do paypal)**, posso trabalhar com idéia de que integer sempre será inteiro e number sempre será decimal com 2 casas, mas precisaria confirmar se existe a situação que vem com o type number e deveria ser inteiro, @leomp12

Reply:
> posso trabalhar com idéia de que integer sempre será inteiro e number sempre será decimal com 2 casas

`integer` sempre será inteiro, `number` pode ser N casas decimais, no caso do `InputMoney` você pode considerar que é só 2 casas (por enquanto) pra ajudar a formatar o input...

Reply:
`InputMoney` pra integer você mocka 0 centavos (`R$ _,00`), e retorna sempre um inteiro